portal/app/farm_updater/egauge_api.py

```diff
@@ -79,7 +78,9 @@ def processing_egauge_data(self):
         from app.models import FarmDevice
         from app.models import DeviceType
 
-        # self.delete_eguage_db_entries()
+        if self.db_delete:
+            self.delete_eguage_db_entries()
```

Comment:
in the delete db entries, it's worth also filtering by home id before the delete operation.